### ISSUE TITLE
"Duplicate Reads" metric not calculated, always outputs 0.

### ISSUE THREAD

**Issue by kachulis** [getzlab/rnaseqc#60]
The "Duplicate Reads" metric is added to the output, however it is never actually calculated.  Since there is also a "Mapped Duplicate Reads" metric which is calculated, it would seem reasonable to remove the "Duplicate Reads" metric

**Comment by agraubert** [comment]
You're absolutely correct. This seems to be a leftover counter from earlier in development. Thanks for pointing this out
